fred2/sexp_tree.cpp

```diff
@@ -7348,6 +7356,25 @@ sexp_list_item *sexp_tree::get_container_multidim_modifiers(int con_data_node) c
 	return head.next;
 }
 
+sexp_list_item* sexp_tree::check_for_dynamic_sexp_enum(int opf)
+{
+	sexp_list_item head;
+
+	int item = opf - First_available_list_id;
+
+	if (item <= (int)Dynamic_enums.size()) {
```

Comment:
this must be a true <, not a <=.
accessing element i from a vector with size i will cause a CTD